railties/test/generators/actions_test.rb

```diff
@@ -150,16 +147,13 @@ def test_environment_with_block_should_include_block_contents_in_environment_ini
   end
 
   def test_git_with_symbol_should_run_command_using_git_scm
-    @mock_generator.expect(:call, nil, ['git init'])
-    generator.stub(:run, @mock_generator) do
+    assert_called_with(generator, :run, ['git init']) do
       action :git, :init
     end
   end
 
   def test_git_with_hash_should_run_each_command_using_git_scm
-    @mock_generator.expect(:call, nil, ["git rm README"])
-    @mock_generator.expect(:call, nil, ["git add ."])
-    generator.stub(:run, @mock_generator) do
+    assert_called(generator, :run, times: 2) do
```

Comment:
In my opinion this kind of test refactoring open potential holes in our suite. The assertions used to be specific, now with `times: 2` they are very generalized and don't have much meaning left.